trade-finance-manager-api/src/v1/controllers/user/user.controller.js

```diff
@@ -17,6 +18,28 @@ exports.findOne = async (_id, callback) => {
   collection.findOne({ _id: { $eq: ObjectId(_id) } }, callback);
 };
 
+exports.findByEmails = async (emails) => {
+  const collection = await db.getCollection('tfm-users');
+  const emailsRegex = emails.map(email => new RegExp(`^${email}$`, 'i'));
+  const users = await collection.find({ 'email': { $in: emailsRegex }}).toArray();
```

Comment:
Need to add some documentation here, also could the email regex be moved into a helper function?